docs-source/documentation/advanced-scenarios/browsers.md

```diff
@@ -0,0 +1,50 @@
+# Browsers
+
+The SDK is configured to use a platform specific browser implementation to handle the redirect to Auth0. Depending on the platform, one or more Browser implementations might be available.
+An example is the Auth0.OidcClient.iOS SDK, which auto selects the browser based on the version of iOS used.
+
+Even though we do not recommend changing this, you could ensure all versions of iOS use the same browser by passing an explicit browser instance to the Browser property of the `Auth0ClientOptions`:
```

Comment:
Why is this not recommended? If there's a recommendation against doing this, we should explain why.

Or is this more of a "you will not normally need to do this"?

Reply:
```suggestion
Even though we do not recommend changing this, you could ensure all versions of iOS use the same browser by passing an explicit browser instance to the `Browser` property of the `Auth0ClientOptions`:
```

Reply:
> Why is this not recommended? If there's a recommendation against doing this, we should explain why.

Auto Select ensures it uses the best available option based on the iOS version. Changing that doesnt sounds like something to recommend. So I think it comes down to "You will not normally need this", let me rephrase that!